CMakeLists.txt

```diff
@@ -167,3 +160,24 @@ function(mbed_generate_bin_hex target)
         VERBATIM
     )
 endfunction()
+
+#
+# Configure Mbed OS CMake application (default).
```

Comment:
```suggestion
# Configure an Mbed OS application.
```
As it is clearly CMake there is no point mentioning it.
I do not quite understand what you mean by `(default)` , can you explain?

Reply:
That this is default one provided by Mbed but can be overwritten (something else used). I can remove default or should we rephrase?